public/docs/_examples/server-communication/dart/lib/toh/hero_list_component.dart

```diff
@@ -0,0 +1,50 @@
+// #docregion
+import 'package:angular2/angular2.dart';
+import 'hero.dart';
+import 'hero_service.dart';
+
+@Component(
+    selector: 'hero-list',
+// #docregion template
+    template: '''
+  <h3>Heroes:</h3>
+  <ul>
+    <li *ngFor="#hero of heroes">
+      {{ hero.name }}
```

Comment:
In general, we don't use spaces, so let's be consistent here and write it as '{{hero.name}}'.